### PR TITLE
17791 Improve contrast of details pane watch/star icon

### PR DESCRIPTION
**Pull request by myabc**
https://community.openproject.org/work_packages/17791

### CODE REVIEW

**Comment by TeatroIO** [comment]
I've prepared a stage to preview changes. <a href='http://myabc-fix-17791-watch-star-icon-contrast.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com' target='_blank'>Open stage</a> or <a href='http://myabc-fix-17791-watch-star-icon-contrast.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com/teatro.log' target='_blank'>view logs</a>.

**Comment by myabc** [comment]
https://community.openproject.org/work_packages/17791

File: app/assets/stylesheets/content/_work_packages.sass [hunk 1]
```diff
@@ -98,22 +98,6 @@ i
...
-    .star
-      padding: 0 2px 0 0
-      vertical-align: -3px
-      color: #ccc
-      margin: 0 0 0 -2px
-      cursor: pointer
-      &:hover, &.active
-        color: #f8d033
-      &.active:hover
-        color: #ccc
```
Comment (ulferts) [inline app/assets/stylesheets/content/_work_packages.sass:115]: :+1:

**Comment by ulferts** [comment]
@floriank can you please take a look at this one?

**Pull request merged**